enrollment-server/src/main/java/com/wultra/app/enrollmentserver/configuration/OpenApiConfiguration.java

```diff
@@ -49,16 +50,28 @@
 )
 public class OpenApiConfiguration {
 
+    private static final String PACKAGE_ADMIN = "com.wultra.app.enrollmentserver.controller.api.admin";
+
     @Bean
     public GroupedOpenApi defaultApiGroup() {
-        String[] packages = {
+        final String[] packages = {
                 "io.getlime.security.powerauth",
                 "com.wultra.app.enrollmentserver.controller.api"
         };
 
         return GroupedOpenApi.builder()
                 .group("enrollment-server")
                 .packagesToScan(packages)
+                .packagesToExclude(PACKAGE_ADMIN)
+                .build();
+    }
+
+    @Bean
+    @ConditionalOnProperty(value = "enrollment-server.admin.enabled", havingValue = "true")
+    public GroupedOpenApi adminApiGroup() {
+        return GroupedOpenApi.builder()
+                .group("enrollment-server-admin")
```

Comment:
Admin controller is not shown on the default swagger page, but you may see it if you want to.